### PR TITLE
Roadmap

### PR DESCRIPTION
**Pull request by sukhrajghuman**
https://designsystem-featureroadmap-test.apps.y.cld.gov.au/roadmap/

user and pass: ds

### CODE REVIEW

File: content/roadmap/now.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+## Now
+
+Things we are working on currently:
+
+- Form component that details guidance around form design, validation and error text.
+- Full page form template example
+- Add more detailed documentation around existing templates
+- Automate documentation workflow
```
Comment (adamzerella) [inline content/roadmap/now.md]: @sukhrajghuman Maybe we should reword this to `Improve component architecture to allow for extendability, performance and automation` ? 

Maybe that can just be another point(s)

File: content/roadmap/now.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+- Improve offline experience
```
Comment (adamzerella) [inline content/roadmap/now.md]: Nice 👍

File: content/roadmap/next.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+Things we will be working on in the near future:
+
+- Build most requested components from the community
+- Decouple components from one another (e.g. Body)
+- Sourcing contributions from the community for an Angular implimentation of the Design System
+- Make it easier to get started with the Design System
+    - Run a workshop on getting started with the Design System for innovation month
+    - Video tutorials on getting started with the Design System
+- Make it easier to contribute to the Design System
+    - Speed up deployment piplelines and install times for components
```
Comment (adamzerella) [inline content/roadmap/next.md]: Already done? Should be in `Now` ?

File: content/_shared/footer.md [hunk 4]
```diff
@@ -5,7 +5,7 @@ mainlist:
   - text: Forums
     link: https://community.digital.gov.au/c/designsystem
   - text: Roadmap
-    link: https://github.com/orgs/govau/projects/7?fullscreen=true
```
[the remark this answers is not in the document]
Comment (sukhrajghuman) [reply content/_shared/footer.md]: yes, must have been merge conflict
